### PR TITLE
[Firefox] Set partial implementation for grid-auto-columns/rows

### LINKED ISSUE #2782: CSS grid-auto-rows not fully supported by latest Firefox
The [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-rows#Browser_Compatibility) on grid-auto-rows specifies that it's fully supported by Firefox. This is not true. There's a [2 year old bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1339672) that has not been fixed yet that causes Firefox to ignore the `grid-auto-rows` value if you specify more than one track.

Can we update the browser compatibility to indicate that the support is partial?

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
Fixes #2782.  See https://bugzilla.mozilla.org/show_bug.cgi?id=1339672 for more information.  (Confirmed it's still a bug.)

### CODE REVIEW

File: css/properties/grid-auto-columns.json [hunk 1]
```diff
@@ -48,10 +48,14 @@
             },
             "firefox": [
               {
-                "version_added": "52"
+                "version_added": "52",
+                "partial_implementation": true,
+                "notes": "Does not accept multiple track-size values.  See <a href='https://bugzil.la/1339672'>bug 1339672</a>."
```
Comment (ddbeck) [inline css/properties/grid-auto-columns.json]:
```suggestion
                "notes": "Does not accept multiple track-size values. See <a href='https://bugzil.la/1339672'>bug 1339672</a>."
```

File: css/properties/grid-auto-columns.json [hunk 2]
```diff
@@ -48,10 +48,14 @@
...
               },
               {
                 "version_added": "40",
+                "partial_implementation": true,
+                "notes": "Does not accept multiple track-size values.  See <a href='https://bugzil.la/1339672'>bug 1339672</a>.",
```
Comment (ddbeck) [inline css/properties/grid-auto-columns.json]:
```suggestion
                "notes": "Does not accept multiple track-size values. See <a href='https://bugzil.la/1339672'>bug 1339672</a>.",
```

File: css/properties/grid-auto-columns.json [hunk 3]
```diff
@@ -63,10 +67,14 @@
             ],
             "firefox_android": [
               {
-                "version_added": "52"
+                "version_added": "52",
+                "partial_implementation": true,
+                "notes": "Does not accept multiple track-size values.  See <a href='https://bugzil.la/1339672'>bug 1339672</a>."
```
Comment (ddbeck) [inline css/properties/grid-auto-columns.json]:
```suggestion
                "notes": "Does not accept multiple track-size values. See <a href='https://bugzil.la/1339672'>bug 1339672</a>."
```

File: css/properties/grid-auto-columns.json [hunk 4]
```diff
@@ -63,10 +67,14 @@
...
               },
               {
                 "version_added": "40",
+                "partial_implementation": true,
+                "notes": "Does not accept multiple track-size values.  See <a href='https://bugzil.la/1339672'>bug 1339672</a>.",
```
Comment (ddbeck) [inline css/properties/grid-auto-columns.json]:
```suggestion
                "notes": "Does not accept multiple track-size values. See <a href='https://bugzil.la/1339672'>bug 1339672</a>.",
```

**Comment by vinyldarkscratch** [comment]
Haha, you're all good @ddbeck -- please, bring on the nitpicking!  XP

Notes have been fixed.  :wink:

**Pull request merged**